charts/opensearch/values.yaml

```diff
@@ -324,6 +324,37 @@ fullnameOverride: ""
 
 masterTerminationFix: false
 
+probes:
+  startup:
+    enabled: true
+    interval: 5
+    delay: 15
+  # Enabled by default, utilizes tcpSocket to check the transport port of the pod
+  liveness:
+    enabled: true
+    interval: 10
+  # ensure the username and password used for READINESS_USERNAME and READINESS_PASSWORD has permissions to access /_cat/health otherwise the pod will never start
+  readiness:
+    enabled: false
+    interval: 10
+    delay: 30
+    # Default utilizes READINESS_USERNAME and READINESS_PASSWORD to query _cat/health for status
+    default: true
+    username: admin
+    password: admin
```

Comment:
Does the demo not expose a user that can do health endpoints checks? 

I know this is something users should actively resolve themselves. But I can see this scenario:

* User has their own security config / Uses defaults BUT changes admin
* Has their own parent chart without looking at full values
* Node never ready because they haven't updated this field with their new securityConfig credentials

Is there anyway we can template this based on an existing set of values? I had a quick look. I don't think there is.

Reply:
I guess the default accepts all states anyway, so maybe this is a non-issue.

Reply:
The default accepts all to avoid that specific issue, though they should definitely configure it correctly for their environment, if there is a larger meta-chart that defines good practices for a full deploy (master, client, data, dashboards) perhaps we set the defaults with a dedicated readiness probe user there?

Reply:
The other reason I think we ought to leave it as it is, is that is the users are not using a declarative setup and are instead using api calls to configure users they would then need to be able to start the first time without requiring a specific user and then redeploy with updated probes. I don't know how common that use case is, but I imagine it is at least some of the deployments.